source/analysis/AM2DAdditionAB.cpp

```diff
@@ -250,5 +254,6 @@ void AM2DAdditionAB::computeCachedValues() const
             cachedData_[j] += data.at(j);
     }
 
+    cachedDataRange_ = AMUtility::rangeFinder(cachedData_);
```

Comment:
I like the idea of collecting together 'misc' static functions into this common AMUtility class. NIce!